### PR TITLE
Fixing group errors when phoneNumber has the 9 digit

### PR DESCRIPTION
**Pull request by fernanda-di-queiroz-f**
Fixes # .

## Changes proposed in this pull request

- Fixing result formatting error when phoneNumber has the 9 digit 

To test (it takes a while): `npm install github:<username>/venom#<branch>`

### CODE REVIEW

File: src/lib/wapi/validation/group.js [hunk 1]
```diff
@@ -90,3 +90,9 @@ export const getAddParticipantStatusError = (statusError) => {
       return GROUP_ERRORS.FORBIDDEN
   }
 }
+
+export const normalizePhoneNumber = (phoneNumber) => {
```
Comment (CBeraldo) [inline src/lib/wapi/validation/group.js:94]: Basear no whatsapp-server temos esse mesmo problema.

**Review by CBeraldo — CHANGES_REQUESTED** [review]

File: src/lib/wapi/validation/group.js (hunk 1 above)
Comment (fernanda-di-queiroz-f) [reply src/lib/wapi/validation/group.js]: Ajustado

File: src/lib/wapi/validation/group.js [hunk 2]
```diff
@@ -90,3 +91,37 @@ export const getAddParticipantStatusError = (statusError) => {
       return GROUP_ERRORS.FORBIDDEN
   }
 }
+
+export const normalizePhoneNumber = (contactPhoneNumber) => {
```
Comment (CBeraldo) [inline src/lib/wapi/validation/group.js:95]: Criar novo arquivo

File: src/lib/wapi/functions/remove-participant.js [hunk 3]
```diff
@@ -34,9 +38,14 @@ export async function removeParticipant(groupId, contactsId) {
   }
   requestResult.participants.forEach((participant) => {
     const phoneNumber = participant.userWid._serialized
```
Comment (CBeraldo) [inline src/lib/wapi/functions/remove-participant.js:40]: const index = getContactIndex(phoneNumber, contacts)

**Review by CBeraldo — CHANGES_REQUESTED** [review]

**Pull request merged**